website/content/docs/integrations/consul-integration.mdx

```diff
@@ -93,16 +262,45 @@ remaining fields are required to match those shown here.
 }
 ```
 
-</Tab>
-<Tab heading="Consul Enterprise">
+</CodeBlockConfig>
+
+The `JWKSURL` address must be reachable by all Consul servers and should
+resolve to multiple Nomad agents to avoid a single point of failure. Both Nomad
+servers and clients are able to handle this request.
+
+[![Consul Auth Method][img_consul_auth_method]][img_consul_auth_method]
+
+When Nomad is configured to use mTLS, [`tls.verify_https_client`][] must be set
+to `false` since it is not possible to provide client certificates to the
+Consul auth method. Alternatively, you may expose Nomad's JWKS URL from a proxy
+or load balancer that terminates the TLS connection.
+
+If the Consul servers are not able to reach Nomad's JWKS URL, the public keys
+may be read from Nomad's `/.well-known/jwks.json` and provided to the auth
+method directly using the [`JWTValidationPubKeys`][] parameter. The keys must
```

Comment:
The passive voice makes this ambiguous. I would word this as "you may read the public keys..." so that it's clear this is something that you, the cluster admin, needs to do manually.

(Same for this paragraph's twin in the Vault doc)

Reply:
I tried to keep this short since it's in the middle of a long text, but it's important information to have, so I moved this and the mTLS discussion to their own section:
https://nomad-git-docs-consul-vault-integrations-hashicorp.vercel.app/nomad/docs/integrations/consul/acl#important-considerations-about-the-jwks-url

This allows us to expand them as necessary.